app/ui/parse_resume.py

```diff
@@ -142,7 +171,7 @@ def match_to_jobs(self, skills, num_jobs=3, skills_per_job=10):
 
             # Grab the relevant information to serve back
             prediction = {
-                "job_name": self.topic_names[topic_number],
+                "job_name": self.topic_names[topic_number + 1],
```

Comment:
this doesn't make any sense. The `topic_number` is coming directly from the LDA model, so I don't get why we'd have to do this adjustment.

If the keys in `topic_names` are offset by one from the LDA model, we should just make `topic_names` line up with the LDA model

Reply:
well, that was exactly how the code was before you changed it to using without a `+1` and it started showing weird results. I think @samgoodgame wrote that original code and may have more context on why we need a `+1`.
If the LDA has the topic numbers starting at `0` and `topic_names` has them starting at `1` - it makes sense. We can change either of them to do the correct thing - but as my comment in the PR says, this is a temporary fix to get us back to a working end-end